tests/test_wsclient.py

```diff
@@ -53,7 +53,7 @@ def test_session_in_headers(self):
 
         self.assertTrue('headers' in content)
         self.assertIn('cookie', [x[0] for x in content['headers']])
-        self.assertIn(b'sessionid', [x[1] for x in content['headers'] if x[0] is 'cookie'])
```

Comment:
Please provide me with content value and I'll try to improve this.

Reply:
this is what i got.

{'reply_channel': 'defaultqFyJJ', 'path': '/', 'headers': [['cookie', b' sessionid=chn75dd650d27ba86a8f0f0617567dd9']]}